ebuild-writing/eapi/text.xml

```diff
@@ -616,6 +616,771 @@ installation targets.
   </dd>
 </dl>
 
+</body>
+</subsection>
+</section>
+
+<section>
+<title>EAPI 8</title>
+<body>
+
+<note>
+This section is based on
+<uri link="https://mgorny.pl/articles/the-ultimate-guide-to-eapi-8.html">
+The ultimate guide to EAPI 8</uri> by Michał Górny.
+</note>
+
+</body>
+
+<subsection>
+<title>EAPI 8 tree layout</title>
+<body>
+
+<dl>
+  <dt>Less strict naming rules for updates directory</dt>
+  <dd>
+    <p>
+    Up to EAPI 7, the files in the <c>profiles/updates</c> directory had to
+    follow strict naming by quarters like <c>2Q-2021</c>, indicating the
+    quarter and the year when they were added. Such a choice of name had the
+    side effect that lexical sorting of filenames was quite inconvenient.
```

Comment:
_unsuitable_ maybe? The way I understand it, it was quite convenient to sort lexicographically, it just wasn't very meaningful to us.